onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -115,6 +128,19 @@ BasicBackend::BasicBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
   if ((global_context_.use_compiled_network == true && import_blob_status == false) || vpu_status == false) {
     if(!openvino_ep::backend_utils::UseCompiledNetwork()) {
       ie_cnn_network_ = CreateCNNNetwork(model_proto, global_context_, subgraph_context_, const_outputs_map_);
+
+    #if defined(OPENVINO_2022_1)
+      // Replace framework output names with OV output names
+      std::unordered_map<std::string, int> new_names;
+      for(auto output_name: subgraph_context_.output_names)
+      {
```

Comment:
Please add some comments as to the reason for this change

Reply:
Done